packages/discord.js/src/structures/GuildMember.js

```diff
@@ -263,7 +263,11 @@ class GuildMember extends Base {
    * @readonly
    */
   get moderatable() {
-    return this.manageable && (this.guild.me?.permissions.has(PermissionFlagsBits.ModerateMembers) ?? false);
+    return (
+      this.manageable &&
+      !this.permissions.has(PermissionFlagsBits.Administrator) &&
```

Comment:
Shouldn't this check be done before the rest instead? (would make it a faster path for members that have Admin perm)

Reply:
Oh I see, I changed that